docs/user-guide/plugins/custom-email-notifier.md

```diff
@@ -0,0 +1,56 @@
+# Custom Email Notifier
+
+## Introduction
+The **Custom Email Notifier** plugin by Devtron enables user to set an automated email notifications to respective stakeholders of that specific CI/CD workflow. The Custom Email Notifier can be integrated into pre/post stages of Devtron CI/CD workflows to keep team members informed about current status and other pipeline events. 
```

Comment:
Why will I use this and not native notifications from global configurations?

Reply:
Add usecases like, configure notification at end of job, or based on certain output of your plugins of pre-post cd, or in autoremediation